vars/standardReleasePipeline.groovy

```diff
@@ -40,7 +40,6 @@ void call(Map args = [:], Closure body) {
             always {
                 script {
                     postCleanup()
-                    sh 'docker image prune -f --all'
```

Comment:
Why remove this?

Reply:
Caused an error looking for docker command on agent node.

Reply:
Will this affect any other jenkinsfile that calls this?
Ex: We want to do the cleanup for docker build.

Reply:
No, this file is not used anywhere today.

Reply:
shouldn't agent node supposed to have docker installed? @gaiksaya

Reply:
@rishabh6788 this file is using docker container running on a jenkins linux agent.
The agent can have docker but the container doesnt have to.
Except docker builder.

Reply:
understood, thank you!